src/domains/ecommerce-common/00_alerts.tf

```
@@ -48,14 +48,23 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_for_checkout_a
   description    = "eCommerce Availability less than or equal 99%"
   enabled        = true
   query = (<<-QUERY
+let thresholdTrafficMin = 40;
+let thresholdTrafficLinear = 100;
+let lowTrafficAvailability = 80;
```

```suggestion
let lowTrafficAvailability = 90;
```
what about set this threshold to the one actually used for alert?

---

I would set this lowTrafficAvaliability to 90 also for all other alerts as for now, and perform tuning if needed in future, what do you think?